nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/source/PasswordAliasConfigSource.java

```diff
@@ -86,6 +89,20 @@ public int getOrdinal() {
         return properties;
     }
 
+    @Override
+    public Set<String> getPropertyNames() {
+        Set<String> propertyNames = new HashSet<>();
+        if (store != null) {
+            Iterator<String> keys = store.keys();
```

Comment:
This can be converted into Java 8 Stream construct instead of using raw iterators.
Code smell

Reply:
Call me old fashioned I still love raw iterators

Reply:
Yea you are old fashioned. So am I :)

Reply:
Just to make sure
Do you refer to
```
store.keys().forEachRemaining((key) -> propertyNames.add(key));
```
or 
```
Stream<String> stream = StreamSupport.stream(Spliterators.spliteratorUnknownSize(store.keys(),Spliterator.ORDERED), false);
stream.forEach(key -> propertyNames.add(key));
```

getproperties() method also use raw iterator. Do I change this too?